text/src/autogluon/text/automm/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -95,6 +101,12 @@ model:
     insert_sep: False
     text_segment_num: 1
     stochastic_chunk: False
+    text_detection_length: 10
```

Comment:
Why do we choose 10? Does it mean 10 words?

Reply:
For tabular data, can we assume each categorical and numerical value, e.g, 3, is only one word?

Reply:
I think we need a better key name to make users easier to get what is used for. `text_detection_length` tells nothing about the minimum word number to perform augmentation.

Reply:
My understanding is that  when tabular feature are used text, it will also have column name eg: age 3. Column name maybe long. But I put this in config such that it can be specified given datasets.